lib/bitex_bot/models/api_wrappers/api_wrapper.rb

```diff
@@ -19,8 +19,8 @@ class ApiWrapper
     :timestamp, # Integer
     :order # Actual order object
   ) do
-    def method_missing(method_name, *args, &_block)
-      order.send(method_name, *args, &_block) || super
+    def method_missing(method_name, *args, &block)
```

Comment:
This is broken. Take a look at https://stackoverflow.com/questions/4683844/ruby-how-do-i-pass-all-parameters-and-blocks-received-by-one-method-to-another
We should instruct CircleCI to run tests first and rubocop after.

Reply:
fixed